hypervisor/arch/x86/mtrr.c

```diff
@@ -145,11 +145,13 @@ static uint32_t update_ept(struct vm *vm, uint64_t start,
 	case MTRR_MEM_TYPE_UC:
 	default:
 		attr = EPT_UNCACHED;
+		break;
 	}
 
-	ept_mr_modify(vm, (uint64_t *)vm->arch_vm.nworld_eptp,
-			start, size, attr, EPT_MT_MASK);
-	return attr;
+	if (ept_mr_modify(vm, (uint64_t *)vm->arch_vm.nworld_eptp,
+				start, size, attr, EPT_MT_MASK) != 0) {
+		pr_err("%s: Modify memory attribute failed", __func__);
+	}
 }
```

Comment:
Just realized why modify_or_del_pdpte/mmu_modify_or_del  ... returns error... they should success...
let us wait @lifeix  to be back and discuss.